CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ include_directories(Combat/Include)
 add_subdirectory(Combat/Source)
```

Comment:
Cała koncepcja jest wykonana przy dziedziczeniu, ale bez żadnego pomysłu i zasad programowania obiektowego. Kod napisany bez przemyślenia i ustaleń ---> dlatego 90% tego jest do wyrzucenia. Tak jak mówiłem, ustalcie pierw na slacku to będzie można to sprawdzić zanim zaczniecie pisanie kodu....


Deck powinno implementować:
- dobierz kartę (z argumentu funkcji)
- odrzuć kartę (zwróć wskaźnik na tą kartę)
- przetasuj karty 
i inne podobne funkcje, jeśli byłyby używany

TableDeck jeśli jest implementacją tego co leży na stole i gra to jest do wyrzucenia, od tego powinna być MapaStołu, bo te karty są odsłonięte i każdy może je przeglądać + one grają.

Kolejne HandDeck, DiscardDeck powinny tak używać funkcja klasy głównej by tylko je nadpisywać lub używać w swoich funkcjach by nie powtarzać kodu. Dodatkowo te deck powinny mieć spójny interfejs.